### PR TITLE
feat(deadline): add content rendering for index and detail and toggle push device assignment

### PR DESCRIPTION
**Pull request by donni106**
SVA-710

### CODE REVIEW

**Review by ardasnturk — CHANGES_REQUESTED** [review]
I just haven't tried the notification feature, but everything else is fine.

File: src/types/GenericType.ts [hunk 1]
```diff
@@ -1,5 +1,6 @@
 export enum GenericType {
   Commercial = 'Offer',
   Job = 'Job',
-  Noticeboard = 'Noticeboard'
+  Noticeboard = 'Noticeboard',
+  Deadline = 'Deadline'
```
[the remark this answers is not in the document]
Comment (donni106) [reply src/types/GenericType.ts:5]: i agree. i sorted everywhere else alphabetically and missed it here.

File: src/screens/BookmarkScreen.js [hunk 2]
```diff
@@ -113,6 +115,16 @@ export const BookmarkScreen = ({ navigation }) => {
...
+        {getSection(
+          QUERY_TYPES.GENERIC_ITEMS,
+          getGenericItemSectionTitle(GenericType.Noticeboard),
+          GenericType.Noticeboard
+        )}
+        {getSection(
+          QUERY_TYPES.GENERIC_ITEMS,
+          getGenericItemSectionTitle(GenericType.Deadline),
+          GenericType.Deadline
+        )}
```
[the remark this answers is not in the document]
Comment (donni106) [reply src/screens/BookmarkScreen.js:127]: yes

File: src/types/GenericType.ts (hunk 1 above)
[the remark this answers is not in the document]
Comment (ardasnturk) [reply src/types/GenericType.ts]: https://github.com/ikuseiGmbH/smart-village-app-app/pull/707/commits/b31209f4bb845b19151ca458c28387d3834c9999

File: src/screens/BookmarkScreen.js (hunk 2 above)
[the remark this answers is not in the document]
Comment (ardasnturk) [reply src/screens/BookmarkScreen.js]: https://github.com/ikuseiGmbH/smart-village-app-app/pull/707/commits/b31209f4bb845b19151ca458c28387d3834c9999

**Review by ardasnturk — APPROVED** [review]

**Pull request merged**